python/run-tests.py

```diff
@@ -57,6 +57,10 @@ def print_red(text):
 FAILURE_REPORTING_LOCK = Lock()
 LOGGER = logging.getLogger()
 
+PYTHON_MAJOR_VERSION = 3
+PYTHON_MINOR_VERSION = 6
+PYTHON_MICRO_VERSION = 0
```

Comment:
honestly, i don't like this anti-pattern but i'm not sure where else to put it...